contracts/contracts/token/WOETH.sol

```diff
@@ -31,11 +43,40 @@ contract WOETH is ERC4626, Governable, Initializable {
         OETH(address(asset())).rebaseOptIn();
     }
 
-    function name() public view virtual override returns (string memory) {
+    function initialize2() external onlyGovernor {
```

Comment:
Perhaps we should have `initialize()` call `initialize2()`. This way new contract deploys don't need to call both, and we are less likely to make a mistake.